server.js

```diff
@@ -75,29 +92,43 @@ app.post('/transactions/send', (req, res) => {
     const isTransactionValid = burgerNode.addPendingTransaction(req.body);
     if (isTransactionValid) {
         burgerSync.broadcastNewTransaction(transaction);
-        res.send("Transaction accepted!");
+        res.json({
+            "transactionDataHash":transaction.transactionDataHash
+        }).status(200);
     } else {
-        res.send("TRANSACTION REJECTED");
+        res.json({
+            "errorMsg":"Invalid Transaction"
+        }).status(404);
     }
 })
 
 app.get('/address/:address/balance', (req, res) => {
     const address = req.params.address;
+    
+        const safeBalance = burgerNode.getSafeBalanceOfAddress(address);
+        const confirmedBalance = burgerNode.getConfirmedBalanceOfAddress(address);
+        const pendingBalance = burgerNode.getPendingBalanceOfAddress(address);
+
+        res.json({
+            safeBalance,
+            confirmedBalance: parseFloat(confirmedBalance),
+            pendingBalance
+        }).status(200);
+    
 
-    const safeBalance = burgerNode.getSafeBalanceOfAddress(address);
-    const confirmedBalance = burgerNode.getConfirmedBalanceOfAddress(address);
-    const pendingBalance = burgerNode.getPendingBalanceOfAddress(address);
-
-    res.json({
-        safeBalance,
-        confirmedBalance: parseFloat(confirmedBalance),
-        pendingBalance
-    });
 })
 
 app.get('/address/:address/transactions', (req, res) => {
     const address = req.params.address;
-    res.json(burgerNode.getTransactionsOfAddress(address));
+    if(burgerNode.chain.getTransactionsOfAddress(address).transactions.length > 0){
```

Comment:
`burgerNode.chain.getTransactionsOfAddress(address)` is undefined.

Maybe you meant: `burgerNode.getTransactionsOfAddress(address)`